redbox-review/README.md

```diff
@@ -0,0 +1,101 @@
+# Redbox
+
+Library simplifying data transport to Redshift via straighforward configuration and intuitive methods (Pack and Ship).
+
+Redbox is transactional. User packs data until they're finished, after which a call to Ship transports the data to Redshift.
+
+See below and [the Godocs](https://godoc.org/github.com/cgclever/redbox) for API documentation
+
+## Redbox - The Configuration
+
+### RedshiftConfiguration
+
+```
+type RedshiftConfiguration struct {
+  Host              string
+  Port              string
+  User              string
+  Password          string
+  Database          string
+  ConnectionTimeout int    // Defaults to 10 seconds
+}
+```
+
+### NewRedboxOptions
+
+```
+type NewRedboxOptions struct {
+  // Required inputs
+  Schema                string
+  Table                 string
+  S3Bucket              string
+  Truncate              bool
+  RedshiftConfiguration RedshiftConfiguration
+
+  // Optional region of the S3Bucket. If not provided Redbox attempts to use 
+  // the AWS API to get its location, however requires the user have permission for this action.
+  S3Region string
```

Comment:
I would make the case that `S3Region` shouldn't live within the Redshift configuration. The reason is because the region refers to that of the `S3bucket`, not that of the Redshift cluster. Since Redshift can be in one region and `COPY` from another, and the `COPY` command refers to the region of the target bucket, it feels like this should live within the general config, as opposed to the `RedshiftConfiguration`.